rebar.config.script

```diff
@@ -0,0 +1,13 @@
+case <<1:16/native>> of
+    <<1,0>> ->
+        CONFIG;
+    <<0,1>> -> 
+        CXXFLAGS = [ { "CXXFLAGS", "$CXXFLAGS -DWORDS_BIGENDIAN" } ],
+        P = lists:keyfind( port_env, 1, CONFIG ),
+        case P of
+            false ->
+                CONFIG ++ [ { port_env, CXXFLAGS } ];
```

Comment:
Another notice I forgot to add, we don't uses extra spaces between brackets. E.g. this one would be written as `CONFIG ++ [{port_env, CXXFLAGS}];`